common/lib/xmodule/xmodule/js/fixtures/video_yt_multiple.html

```diff
@@ -163,7 +180,14 @@
             <section class="video-player">
               <iframe id="id3"></iframe>
             </section>
+            <a href="#" tabindex="0" class="hide-captions" role="button" aria-disabled="false"
+                  title="${_('Turn off transcripts')}">
+              <i class="icon-caret-right" aria-hidden="true"></i>
```

Comment:
We've recently updated our Font Awesome library, so you'll want to update the icons to use the [latest Font Awesome set] (http://fontawesome.io/cheatsheet/). A rebase against Master might be wise too.